phpseclib/Net/SSH2.php

```diff
@@ -150,6 +150,61 @@ class SSH2
     const READ_NEXT = 3;
     /**#@-*/
 
+    /**#@+
```

Comment:
The `#@+`'s are unnecessary. Their main purpose is to tell phpdoc or whatever to propagate the values down until it encounters a `#@=`.